embedded-kafka/src/main/scala/net/manub/embeddedkafka/EmbeddedKafka.scala

```diff
@@ -271,6 +282,69 @@ sealed trait EmbeddedKafkaSupport {
     }.get
   }
 
+  /**
+    * Consumes the first n messages available in a given topic, deserializing it as a String, and returns
+    * the n messages as a List.
+    *
+    * Only the messsages that are returned are committed if autoCommit is false.
+    * If autoCommit is true then all messages that were polled will be committed.
+    *
+    * @param topic        the topic to consume a message from
+    * @param number       the number of messagese to consume in a batch
+    * @param autoCommit   if false, only the offset for the consumed message will be commited.
+    *                     if true, the offset for the last polled message will be committed instead.
+    *                     Defaulted to false.
+    * @param config       an implicit [[EmbeddedKafkaConfig]]
+    * @param deserializer an implicit [[org.apache.kafka.common.serialization.Deserializer]] for the type [[T]]
+    * @return the first message consumed from the given topic, with a type [[T]]
+    * @throws TimeoutException          if unable to consume a message within 5 seconds
+    * @throws KafkaUnavailableException if unable to connect to Kafka
+    */
+  def consumeNumberMessagesFrom[T](topic: String, number: Int, autoCommit: Boolean = false)(
+    implicit config: EmbeddedKafkaConfig,
+    deserializer: Deserializer[T]): List[T] = {
+
+    import scala.collection.JavaConversions._
```

Comment:
`JavaConversions` is deprecated in 2.12. Could you please use `JavaConverters` instead?